front-end/src/app/reports/form-type-dialog/form-type-dialog.component.html

```diff
@@ -0,0 +1,41 @@
+<p-dialog
+  [style]="{ width: '50%' }"
+  [(visible)]="visible"
+  header="Select form type"
+  [modal]="true"
+  styleClass="p-fluid"
+>
+  <ng-template pTemplate="content">
+    <div class="grid">
+      <div class="col-6">
+        <div class="field">
+          <label for="type">CHOOSE FORM</label>
+          <p-dropdown
+            appendTo="body"
+            [(ngModel)]="selectedType"
+            [options]="formTypeOptions"
+            autoWidth="false"
+            [style]="{ width: '27em' }"
```

Comment:
We might want to move this (and the p-dialog style) into .scss.  Also, does specifying this as a pecentage work (since we're using that on the p-dialog)?